src/database/UIDatabase.ts

```diff
@@ -17,7 +17,56 @@ import { version as appVersion } from '../../package.json';
 
 const { THIS_STORE_NAME_ID, APP_VERSION } = SETTINGS_KEYS;
 
-const translateToCoreDatabaseType = type => {
+export type RealmDatabaseType =
+  | 'ADRForm'
+  | 'ActiveLocation'
+  | 'CashTransaction'
+  | 'CashTransactionName'
+  | 'CashTransactionReason'
+  | 'Customer'
+  | 'CustomerCredit'
+  | 'CustomerInvoice'
+  | 'CustomerTransaction'
+  | 'ExternalSupplier'
+  | 'InternalSupplier'
+  | 'MedicineAdministrator'
+  | 'NegativeAdjustmentReason'
+  | 'OpenVialWastageReason'
+  | 'PCDEvents'
+  | 'Patient'
+  | 'PatientSurveyForm'
+  | 'Payment'
+  | 'Policy'
+  | 'PositiveAdjustmentReason'
+  | 'Prescription'
+  | 'PrescriptionCategory'
+  | 'Provider'
+  | 'Receipt'
+  | 'RequestRequisition'
+  | 'RequisitionReason'
+  | 'ResponseRequisition'
+  | 'Supplier'
+  | 'SupplierCreditCategory'
+  | 'SupplierInvoice'
+  | 'SupplierTransaction'
+  | 'Vaccine'
+  | 'Item';
+
+export type CoreDatabaseType =
+  | 'Transaction'
+  | 'Name'
+  | 'Options'
+  | 'Requisition'
+  | 'InsurancePolicy'
+  | 'InsuranceProvider'
+  | 'TransactionCategory'
+  | 'PatientEvent'
+  | 'Item'
+  | 'FormSchema'
+  | 'MedicineAdministrator'
+  | 'Location';
```

Comment:
Technically all of these within the schema are a core database type https://github.com/openmsupply/mobile/blob/4d3a1ddbb0836003f1cbc59ae4c563f1ba52c321/src/database/schema.js#L196-L259

Reply:
Still this comment to address. Theres a set of object type strings between the two ts types which give errors but are valid i.e `UIDatabase.objects('Stocktake');`

Reply:
I just had a look but I think I lack the knowledge about which table belongs where... Should I create an issue or should we go through it together once you are back in the office?